diff --git a/node_modules/@tamagui/constants/dist/esm/constants.js b/node_modules/@tamagui/constants/dist/esm/constants.js
index aef5893..9fb08db 100644
--- a/node_modules/@tamagui/constants/dist/esm/constants.js
+++ b/node_modules/@tamagui/constants/dist/esm/constants.js
@@ -3,14 +3,7 @@ const isWeb = process.env.TAMAGUI_TARGET === "web";
 const isWindowDefined = typeof window !== "undefined";
 const isServer = isWeb && !isWindowDefined;
 const isClient = isWeb && isWindowDefined;
-const isRSC = process.env.ENABLE_RSC ? (
-  // note this is statically analyzed so no funny business, just access it without optional chaining
-  // @ts-ignore
-  import.meta.env ? (
-    // @ts-ignore
-    import.meta.env.SSR
-  ) : false
-) : false;
+const isRSC = false;
 const idFn = () => {
 };
 const useIsomorphicLayoutEffect = isRSC ? idFn : isServer ? useEffect : useLayoutEffect;
